game-core/src/main/java/games/strategy/triplea/ui/ActionButtons.java

```diff
@@ -253,7 +253,7 @@ public PlaceData waitForPlace(final boolean bid, final IPlayerBridge bridge) {
 
   /** Blocks until the user selects an end-of-turn action. */
   public void waitForEndTurn(final TripleAFrame frame, final IPlayerBridge bridge) {
-    endTurnPanel.waitForEndTurn(frame, bridge);
+    endTurnPanel.waitForDone(frame, bridge);
```

Comment:
'waitForEndTurn' just calls 'waitForDone'. The child implementations in both cases just called their parent class. So we can remove the 'waitForEndturn' method, and remove the 'waitForDone' overrides on the child classes.